Find a functionally equivalent code:android.graphics.Matrix.postSkew:COMMENT
Method Modifier: public      
Comment:/**
 * Postconcats the matrix with the specified skew.
 * M' = K(kx, ky) * M
 */

@@ -1,3 +1,4 @@
 {
-    return native_postSkew(native_instance, kx, ky);
+    native_postSkew(native_instance, kx, ky);
+    return true;
 }

